microshift_release_notes/microshift-4-19-release-notes.adoc

```diff
@@ -27,8 +27,11 @@ This release adds improvements related to the following components and concepts.
 
 //L3 major categories with features in each as L4s
 //[id="microshift-4-19-rhel_{context}"]
-//=== {op-system-base-full}
-//Image mode for RHEL here if GA; otherwise leave in TP
+=== {op-system-base-full}
+
+[id="microshift-4-19-rhel-image-mode_{context}"]
+==== {op-system-base-full} image mode (General Availablity)
```

Comment:
On line 33, should {op-system-base-full} be {op-system-base} since the name is spelled out just above on line 30?